src/armorblox/api/incidents_api.py

```diff
@@ -38,7 +38,9 @@ def list(self, page_token=None, params=None):
         else:
             next_page_token = response_json.get('next_page_token', None)
             total_count = response_json.get('total_count', 0)
-            return response_json.get('incidents', [])
+            incidents = response_json.get('incidents', [])
+            incidents.append({'next_page_token': next_page_token, 'total_count': total_count})
```

Comment:
We should be able to directly return the 2 values in the same call. Easier to use that way.